packages/admin/admin/src/dataGrid/GridCellText.tsx

```diff
@@ -0,0 +1,106 @@
+import { ComponentsOverrides, css, Theme, Typography, useThemeProps } from "@mui/material";
+import React from "react";
+
+import { createComponentSlot } from "../helpers/createComponentSlot";
+import { ThemedComponentBaseProps } from "../helpers/ThemedComponentBaseProps";
+
+export type GridCellTextClassKey = "root" | "hasSecondaryText" | "primaryText" | "secondaryText";
+
+export interface GridCellTextProps
+    extends ThemedComponentBaseProps<{
+        root: "div";
+        primaryText: typeof Typography;
+        secondaryText: typeof Typography;
+    }> {
+    primary: React.ReactNode;
+    secondary?: React.ReactNode;
```

Comment:
Have you considered to also support `children` instead of `primary` and `secondary`? I believe MUI's ListItemText can do both.

Reply:
Good point, will add that 👍🏼